src/webgl/interactions/events/EventManager.ts

```diff
@@ -0,0 +1,41 @@
+import { FederatedPointerEvent } from 'pixi.js'
+import { Container } from 'pixi.js'
+import ApplicationEvents from '../ApplicationEvents'
+import RendererOptions from '../../RendererOptions'
+import Interaction from '../Interaction'
+import HitArea from '../hitArea/HitArea'
+
+export default abstract class EventsManager extends Interaction {
```

Comment:
### EventsManager
- will be the base class for defining things like node events, viewport events, etc.
- simplifies the relationship between hit area + events